### ISSUE TITLE
Add Feed's Folder Picker should match existing iOS one

### ISSUE THREAD

**Issue by vincode-io** [Ranchero-Software/NetNewsWire#2193]
- Should have a title for the page
- Should list containers with indentation when needed
- Should utilize the small icon provider to display an icon next to the item

**Comment by vincode-io** [comment]
![IMG_0547](https://user-images.githubusercontent.com/16448027/86487918-04c9e000-bd25-11ea-8590-baf51cdcc32d.PNG)

**Comment by stuartbreckenridge** [comment]
I will sort the folder picker look/feel when Add Folder functionality is available.

The `NavigationBarTitle` is a bit untidy at this point - it can be changed to _Pick Folder_, but I can't find a way to change it back. 

```swift
var folderPicker: some View {
		Picker("Folder:", selection: $viewModel.selectedFolderIndex, content: {
			ForEach(0..<viewModel.containers.count, id: \.self, content: { index in
				if let containerName = (viewModel.containers[index] as? DisplayNameProvider)?.nameForDisplay {
					if viewModel.containers[index] is Folder {
						Text("\(viewModel.containers[index].account?.nameForDisplay ?? "") / \(containerName)").tag(index)
					} else {
						Text(containerName).tag(index)
					}
				}
			}).navigationBarTitle("Pick Folder") // <-- This changes the title on the folder picker, but there's no way to change it back.
		}).navigationBarTitle("Add Web Feed") // <-- Doesn't work!
	}
```

**Comment by vincode-io** [comment]
Just leave it blank for now.  The picker should have some other way to set it for you, I guess.

**Comment by vincode-io** [comment]
This is looking pretty good, except the icons on the picker should have the accent color applied to them.  I figured, I would just add the modifier and be done with it, but the accent color was wrong in the Picker.  Did you run into this and do you know if this is a known bug?  We might have to file a Feedback for it.

**Comment by stuartbreckenridge** [comment]
@vincode-io This icon colours can be changed by adding `.foregroundColor(.accentColor)`. I was going to commit some code for it, but this is the second time I am seeing crashes when using AppAssets: it refers to `AccountLocalPhone` vs `accountLocalPhone` is the asset catalog. I fixed it before but it seems to have gone back.

![icons](https://user-images.githubusercontent.com/7046652/86544726-fa455b80-bf5b-11ea-8984-0bc63ea0ce2a.png)

**Comment by vincode-io** [comment]
The accent color was the wrong blue when I used `.foregroundColor(.accentColor)`.

If you get a case insensitive error on the Asset Catalog, then rename the item to a completely different name and then back to the correct uppercased name.  When I standardized the Asset Catalog on uppercase names Xcode screwed up the filesystem case sensitivity.  Changing to a completely different name and back again fixes the problem.
